src/applications/gi/containers/Modals.jsx

```diff
@@ -28,6 +29,81 @@ export class Modals extends React.Component {
     );
   }
 
+  productionFlag107() {
```

Comment:
name should be something like `calcBeneficiaryLocationQuestionContent`

Reply:
Updated function name to calcBeneficiaryLocationQuestionContent